### ISSUE TITLE
Java tests failing after arrow upgrade

### ISSUE THREAD

**Issue by pcmoritz** [ray-project/ray#2611]
After upgrading arrow, I'm getting this message:

```
[ERROR] Failed to execute goal on project ray-api: Could not resolve dependencies for project org.ray:ray-api:jar:1.0: Failure to find org.apache.arrow:arrow-plasma:jar:0.10.0-SNAPSHOT in ...
```

see https://travis-ci.com/ray-project/ray/jobs/138952814

@kingchin1218 @salah-man

**Comment by pcmoritz** [comment]
Recently the arrow version number was upgraded from 0.9 to 0.10, could it have anything to do with that?

**Comment by kingchin1218** [comment]
@pcmoritz   Thanks.

**Comment by salah-man** [comment]
We will check and test it as soon as possible.

**Comment by kingchin1218** [comment]
I have tested the branch [pcmoritz's branch](https://github.com/pcmoritz/ray-1/tree/fix-arrow-tensorflow-op).  It worked well.
Note: I have done `make clean` and some `clean_up` as blew:
```
sudo rm -rf python/ray/core
sudo rm -rf python/ray/pyarrow_files
sudo rm -rf thirdparty/build/arrow
sudo rm -rf thirdparty/pkg/arrow
sudo rm -rf thirdparty/build/parquet-cpp
```
@pcmoritz 
How can i reproduce it?
Should i clone it to another dir?

**Comment by kingchin1218** [comment]
It might be that there is not `org.ray` package in [remote repo](https://oss.sonatype.org/content/repositories/snapshots).

**Comment by kingchin1218** [comment]
This issue had been solved in PR[#2607](https://github.com/ray-project/ray/pull/2607).
It could be closed.
@pcmoritz  @robertnishihara
